salesagility/SuiteCRM
# FIXED #7979 - Reports Total table squished

The Report Total field became much smaller at some point and looks weird compared to the main Reports table
## Description




## Motivation and Context

https://github.com/salesagility/SuiteCRM/issues/7979
## How To Test This


  1.  Create a Report in the AOR_Reports module, with a 'total' field.
  2.  View the report.
  3.  See that the total field's table is much smaller than the main table.

## Types of changes


### Final checklist
